### ISSUE TITLE
Completely Locked Out - Unable to Send

### ISSUE THREAD

**Issue by RattitudeMining** [spesmilo/electrum#2062]
Hello everyone,

This is my first post on GitHub but I am in dire need of assistance; I am in hell to say the least.

Too bad we must meet in such crappy circumstances but I don't know what else to do; I need help.  On the wallet, it says to come here for support so, here I am... coming in peace, just want use of my coin again, in the most expeditious manner as possible.

As the title implies, I am unable to send my Bitcoins anywhere at all.  I don't open my wallet often, maybe once a month or so to update but I noticed (today) that version 2.7.11 brought a new type of fee input mechanism and usually doing an update fixes all my headaches.  I tried everything I could think of, down to doing a system restore, alas, my Bitcoin are still trapped in there.

It used to be that I it was giving me a "Invalid Fee" box and wouldn't let me send.  So I switched it from that slider thing, to the manual input version; still no to no avail, it would not send anything.  Then I updated to 2.7.12 and still would not send anything.  Doing a restore did nothing positive.

My knowledge of wallets is still rather slim, this is my first seriously wallet problem, where I need help.  The new fee mechanism is what really killed everything over here.  No matter what I did with it, "invalid fee" message.  I'd rather avoid doing a seed restore entirely, I read somewhere it yielded no result, so why do it.

Maybe I am missing some essential things and update that need updated for Electrum to run like Java or Silverlight?  Will it even run on Windows 7 SP1 for that matter?  I like my operating systems to be as simple as possible.  usually software require certain extensions and to be a certain version.

This is likely going to take some time, right now I am going to be and will be here first thing in AM; can barely type this but wanted to do it before crashing.

Thanks for all and any help in advance!

**Comment by ecdsa** [comment]
your statements are self contradicting. Did you restore from seed or not?
if yes, can you describe what you mean by "did nothing positive" ?

**Comment by RattitudeMining** [comment]
Hello ecdsa,

I did not restore from seed, I did a system restore.
Seed restore is Electrum based, system restore is Windows based.

**Comment by RattitudeMining** [comment]
Actually, I don't believe this and you won't either.
After trying I don't know how many times to send Bitcoin last night using whichever way, I tried for the heck of it this morning, thinking, I'll try again, just for the eff of it see if God loves me and walla!!  It works now!!  Sent a payment and looks like all will be fine for now.

However, the "server defined" tx fee slider needs work.  Seems like no matter what I did, the fee was too high.  In preferences is an option to switch to user defined, that works.

I am so relieved.... I can handle an computer throwing a tantrum, I can't handle stationary coin.... well, not today anyways.

Thank you so much!!

Whatever you did... you're a miracle worker, the issue resolved itself.

Peace out!!

-GR
